forge-gui/src/main/java/forge/util/ImageFetcher.java

```diff
@@ -205,9 +205,10 @@ public void fetchImage(final String imageKey, final Callback callback) {
             }
             final String cardCollectorNumber = paperCard.getCollectorNumber();
             if (!cardCollectorNumber.equals(IPaperCard.NO_COLLECTOR_NUMBER)) {
-                final String scryfallURL = this.getScryfallDownloadURL(paperCard, face, useArtCrop, hasSetLookup, filename, downloadUrls);
-                if (scryfallURL != null && !hasSetLookup)
-                    downloadUrls.add(scryfallURL);
+                if (!hasSetLookup) {
```

Comment:
didn't you change the logic here?
when SetLookup=true no URL will be added now 🤔

Reply:
Yea. I think when i made this change I didn't realize that this function updated downloadUrls and never came back to it.